modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -147,8 +147,10 @@ SiteNavigationAdminManagementToolbarDisplayContext siteNavigationAdminManagement
 	</liferay-ui:search-container>
 </aui:form>
 
-<aui:script require="metal-dom/src/all/dom as dom,frontend-js-web/liferay/modal/commands/OpenSimpleInputModal.es as openSimpleInputModal" sandbox="<%= true %>">
-	var renameSiteNavigationMenuClickHandler = dom.delegate(
+<aui:script require="frontend-js-web/liferay/modal/commands/OpenSimpleInputModal.es as openSimpleInputModal,frontend-js-web/liferay/delegate/delegate.es as delegateModule" sandbox="<%= true %>">
```

Comment:
We can try forwarding this as-is, @kresimir-coko, but there is a non-zero chance that Brian might ask this to be sorted, so FWIW I think you may as well sort it (`delegate`/`delegateModule` comes before `modal`/`openSimpleInputModal`).

Reply:
@wincent Sure, I wasn't sure what the rule here was, I'll push and ping you when it's ready.

Reply:
@wincent I've pushed a commit that sorts it.

Reply:
> I wasn't sure what the rule here was

I don't know if there is a *rule* as such (and we probably don't import multiple modules with `require` all that often), but this just seems like the kind of thing Brian would ask about.

![better-safe-than-sorry](https://user-images.githubusercontent.com/7074/100440373-141b4280-30a5-11eb-9cec-5cc6988cf7be.gif)

Reply:
@kresimir-coko: Can I be really annoying and ask you to drop the `sandbox` attribute as well? I don't think it's required with, er, `require`, as [I commented on another PR](https://github.com/liferay-frontend/liferay-portal/pull/558#discussion_r531502056).

Once you've done that, feel free to hit `ci:forward` and we'll see if the CI smiles upon you.

Reply:
Sure @wincent , I hate it anyway, IDK what it does (I knew at one point) and I just don't like it for some reason, so I'll gladly terminate it.

![](https://media2.giphy.com/media/IZY2SE2JmPgFG/giphy.gif)